Packs/ForescoutEyeInspect/IncidentFields/incidentfield-ForescoutEyeInspect_Source_MAC_Address.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@RazOmry We are trying to minimize the creation of new fields. Please use the common field "Source MAC Address" instead.